python/graphscope/client/session.py

```diff
@@ -1089,6 +1094,64 @@ def get_config(self):
         """Get configuration of the session."""
         return self._config_params
 
+    def backup_vineyard_depployment(self, graphIDs: str, path: str, pvc_name: str):
+        if self._cluster_type != types_pb2.K8S:
+            return
+        # check if session exist
+        if self.session_id is None:
```

Comment:
You could not both to consider such case like the session is invalid or something.